.conda/meta.yaml

```diff
@@ -17,8 +17,10 @@ requirements:
 
   run:
     - python >=3.6
+    - bokeh
     - networkx
     - pandas
+    - pydot
```

Comment:
I also needed to install pygraphviz explicitly, but did not try from scratch.

Reply:
Tested on ubuntu without pygraphviz and the environment from `environment.yml` and it worked.